packages/conventional-recommended-bump/test/index.spec.js

```diff
@@ -77,9 +77,7 @@ describe('conventional-recommended-bump', () => {
 
       const recommendation = await conventionalRecommendedBump({
         cwd: testTools.cwd,
-        preset: {
-          name: 'conventionalcommits'
-        }
+        preset: 'conventionalcommits'
       }, {})
```

Comment:
Why did you change it?

Reply:
Because this string preset causes the issue and performs the same thing as `{  name: 'conventionalcommits'  }`. If you think this is inappropriate, I will write an additional test case for it.